bedrock/firefox/tests/test_redirects.py

```diff
@@ -85,3 +86,437 @@ def test_mobile_app():
+            EXPECTED_FIREFOX_COM_REDIRECT_CODE,
+            False,
+        ),
+        (
+            "/en-US/firefox/features/pinned-tabs/",
+            f"https://www.firefox.com/features/pinned-tabs/{EXPECTED_REDIRECT_QS}",
+            EXPECTED_FIREFOX_COM_REDIRECT_CODE,
+            False,
+        ),
+        (
+            "/en-US/firefox/features/private-browsing/",
+            f"https://www.firefox.com/features/private-browsing/{EXPECTED_REDIRECT_QS}",
+            EXPECTED_FIREFOX_COM_REDIRECT_CODE,
+            False,
+        ),
+        (
+            "/en-US/firefox/features/private/",
+            f"https://www.firefox.com/features/private/{EXPECTED_REDIRECT_QS}",
+            EXPECTED_FIREFOX_COM_REDIRECT_CODE,
+            False,
+        ),
+        (
+            "/en-US/firefox/features/sync/",
+            f"https://www.firefox.com/features/sync/{EXPECTED_REDIRECT_QS}",
+            EXPECTED_FIREFOX_COM_REDIRECT_CODE,
+            False,
+        ),
+        (
+            "/en-US/firefox/features/tips/",
+            f"https://www.firefox.com/features/tips/{EXPECTED_REDIRECT_QS}",
+            EXPECTED_FIREFOX_COM_REDIRECT_CODE,
+            False,
+        ),
+        (
+            "/en-US/firefox/features/translate/",
+            f"https://www.firefox.com/features/translate/{EXPECTED_REDIRECT_QS}",
+            EXPECTED_FIREFOX_COM_REDIRECT_CODE,
+            False,
+        ),
+        (
+            "/firefox/ios/testflight/",
+            f"https://www.firefox.com/channel/ios/testflight{EXPECTED_REDIRECT_QS}",
+            EXPECTED_FIREFOX_COM_REDIRECT_CODE,
+            False,
+        ),
+        (
+            "/firefox/linux/",
+            f"https://www.firefox.com/browsers/desktop/linux/{EXPECTED_REDIRECT_QS}",
+            EXPECTED_FIREFOX_COM_REDIRECT_CODE,
+            False,
+        ),
+        (
+            "/firefox/mac/",
+            f"https://www.firefox.com/browsers/desktop/mac/{EXPECTED_REDIRECT_QS}",
+            EXPECTED_FIREFOX_COM_REDIRECT_CODE,
+            False,
+        ),
+        (
+            "/firefox/more/",
+            f"https://www.firefox.com/more/{EXPECTED_REDIRECT_QS}",
+            EXPECTED_FIREFOX_COM_REDIRECT_CODE,
+            False,
+        ),
+        (
+            "/firefox/set-as-default/",
+            f"https://www.firefox.com/landing/set-as-default/{EXPECTED_REDIRECT_QS}",
+            EXPECTED_FIREFOX_COM_REDIRECT_CODE,
+            False,
+        ),
+        (
+            "/firefox/set-as-default/thanks/",
+            f"https://www.firefox.com/landing/set-as-default/thanks/{EXPECTED_REDIRECT_QS}",
+            EXPECTED_FIREFOX_COM_REDIRECT_CODE,
+            False,
+        ),
+        (
+            "/firefox/switch/",
+            f"https://www.firefox.com/landing/switch/{EXPECTED_REDIRECT_QS}",
+            EXPECTED_FIREFOX_COM_REDIRECT_CODE,
+            False,
+        ),
```

Comment:
```suggestion
```

Reply:
Good spot - this is commented out now in https://github.com/mozilla/bedrock/pull/16356/commits/ecd44e135d6f9adf78155876b1694b177ed06e79

Reply:
@stevejalim this test ^^ still needs removing to pass, along with the one posted below.